CXXFLAGS= # AC_PROG_CXX automatically sets CXXFLAGS to -g -O2 if g++ is the C++
compiler and CXXFLAGS hasn't been set, so by setting CXXFLAGS, we can avoid
this.

I changed that (in the configure script only) to CXXFLAGS="$CXXFLAGS", which
does the same for unset CXXFLAGS but keeps the setting if the variable is set.

Patch from Martin von Gagern <Martin.vGagern@gmx.net>

Index: rezound-0.12.3beta/configure.ac
===================================================================
--- rezound-0.12.3beta.orig/configure.ac
+++ rezound-0.12.3beta/configure.ac
@@ -53,7 +53,7 @@ AC_LANG(C++)
 ############################################################################
 # Checks for programs                                                      #
 ############################################################################
-CXXFLAGS= # AC_PROG_CXX automatically sets CXXFLAGS to -g -O2 if g++ is the C++ compiler and CXXFLAGS hasn't been set, so by setting CXXFLAGS, we can avoid this.
+CXXFLAGS="$CXXFLAGS" # AC_PROG_CXX automatically sets CXXFLAGS to -g -O2 if g++ is the C++ compiler and CXXFLAGS hasn't been set, so by setting CXXFLAGS, we can avoid this.
 AC_PROG_CXX
 AC_PROG_AWK
 AC_PROG_CC
